atsign-foundation/at_mono
# refactor: analysis and docs

**- What I did**

**N.B. : +2000/-2000 is the SVG being regenerated, this PR is not actually that big.**

- Converted my notebook code into a script which automatically rebuilds the package tree diagrams based on the latest info
- Added a composite action to do the above
- Added the composite action to the cron job so that it runs daily with each backup

**- How I did it**

Local changes

**- How to verify it**

See the diagrams that are generated in the final commit on this PR.
The commit should be titled: `docs(ci): Update docs`

**- Description for the changelog**
refactor: analysis and docs
